CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ Changes to Calva.
 
 ## [Unreleased]
 
+- clojure-lsp changes:
+  - Use OS and architecture to identify native binary to download
+  - Fall back to `clojure-lsp-standalone.jar` for environments with no native binary
+  - Use system Java to execute clojure-lsp if a JAR file is provided
+  - Fix: [Calva downloads the wrong clojure-lsp when running in devcontainer
+](https://github.com/BetterThanTomorrow/calva/issues/1598)
+  - Fix: [start clojure-lsp client failed](https://github.com/BetterThanTomorrow/calva/issues/1590)
```

Comment:
We usually only link to the problems fixed. I think we can make an exception here, linking to the PR and with quick access to the issues.

```suggestion
- [Updates to how we choose clojure-lsp executable for different platforms](https://github.com/BetterThanTomorrow/calva/pull/1758), fixes: [#1590](https://github.com/BetterThanTomorrow/calva/issues/1590), and [#1598][#1590](https://github.com/BetterThanTomorrow/calva/issues/1598)
```